### ISSUE TITLE
FR: don't randomize ipv6 ports

### ISSUE THREAD

**Issue by StraightfaceStudios** [tailscale/tailscale#8507]
### What are you trying to do?

IPv6 doesn't use UPnP or NAT-PMP. 

Enabling port randomization shouldn't randomize the ipv6 interface listening port as theoretically every ipv6 device already has a unique non-NAT'ed address and just needs a whitelist in the firewall. 

### How should we solve this?

Leave ipv6 on the default port even if randomize-ports is set in the ACLs or set up two separate ACLs for ipv4 and ipv6. 

### What is the impact of not solving this?

Derpiness

**Comment by DentonGentry** [comment]
The goal is to be able to allow port 41641 to ingress for IPv6 while relying on NAT-PMP for IPv4?

**Comment by StraightfaceStudios** [comment]
Correct, ipv6 should be handled by opening firewall ports manually with a blanket rule or for specific hosts.

Port randomization though for the ipv4 NAT clients dooms any possibility of doing that though.

**Comment by StraightfaceStudios** [comment]
Correct, ipv6 should be handled through manual firewall rules since there are no automatic port opening protocols. So a static port would allow servers and subnet routers to be explicitly opened.
